packages/perseus-core/package.json

```diff
@@ -0,0 +1,23 @@
+{
+    "publishConfig": {
+        "access": "public"
+    },
+    "name": "@khanacademy/perseus-core",
+    "version": "0.0.1",
+    "description": "Core Perseus API (includes renderers and widgets)",
+    "module": "dist/es/index.browser.js",
+    "main": "dist/index.browser.js",
+    "source": "src/index.js",
+    "scripts": {
+        "test": "bash -c 'yarn --silent --cwd \"../..\" test ${@:0} $($([[ ${@: -1} = -* ]] || [[ ${@: -1} = bash ]]) && echo $PWD)'"
```

Comment:
Why the need for bash here?

Reply:
Not sure.  It might an escaping/interpolation issue of some sort.